include/kodlab_mjbots_sdk/pid.h

```diff
@@ -0,0 +1,180 @@
+/**
+ * @file pid.h
+ * @author Chandravaran Kunjeti (kunjeti@seas.upenn.edu)
+ * @brief Class implementation of PID
+ * @date 9/30/2022
+ *
+ * @copyright 2022 The Trustees of the University of Pennsylvania.
+ * All rights reserved.
+ *
+ */
+
+#pragma once
+
+#include <math.h>
+
+#include <vector>
+
+#define HIGH_SATURATION_INDEX_ 0
+#define LOW_SATURATION_INDEX_ 0
+
+namespace kodlab {
+
+  /**
+   * @brief PIDController class that allows you to use PID control.
+   */
+  class PIDController {
+  public:
+    /**
+     * @brief Construct a Pid object
+     * @param p_gain Proportional Gain
+     * @param d_gain Derivative Gain
+     * @param i_gain Integral Gain
+     * @param time_step Time difference between 2 updates
+     */
+    PIDController(double p_gain, double d_gain, double i_gain, double time_step,
+      double deadband = 1.0,
+      std::vector<double> saturation = { INFINITY, -INFINITY })
+      : kp_(p_gain),
+      kd_(d_gain),
+      ki_(i_gain),
+      deadband_(deadband),
+      saturation_(saturation) {
+      time_step_ = time_step;
+    }
```

Comment:
Format per [this style guide section](https://drake.mit.edu/styleguide/cppguide.html#Constructor_Initializer_Lists).